### PR TITLE
HDDS-2747. interface/JavaApi.md

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

interface/JavaApi.md added.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2747

## How was this patch tested?

hugo server -D

### CODE REVIEW

**Comment by iamabug** [comment]
@xiaoyuyao Could you help review this ? Thanks.

File: hadoop-hdds/docs/content/interface/JavaApi.zh.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Java API（测试）" 
+title: "Java API"
 date: "2017-09-14"
 weight: 1
-summary: Ozone has a set of Native RPC based APIs. This is the lowest level API's on which all other protocols are built. This is the most performant and feature-full of all Ozone protocols.
+summary: Ozone 有一套基于原生 RPC 的 API，其它协议都由这个最底层的 API 扩展而来，它也是所有协议中性能最好、功能最全的。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/JavaApi.zh.md]: 它也是所有协议中性能最好、功能最全的。=》它也是所有Ozone支持的协议中性能最好、功能最全的。

File: hadoop-hdds/docs/content/interface/JavaApi.zh.md [hunk 2]
```diff
@@ -21,4 +21,132 @@ summary: Ozone has a set of Native RPC based APIs. This is the lowest level API'
   limitations under the License.
 -->
 
-测试页面
+Ozone 自带了支持 RPC 的客户端库，对于一般的应用场景也可以使用兼容 S3 的 REST 接口替代 Ozone 客户端 API。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/JavaApi.zh.md]: 兼容 S3 =兼容 AWS S3

File: hadoop-hdds/docs/content/interface/JavaApi.zh.md [hunk 3]
```diff
@@ -21,4 +21,132 @@ summary: Ozone has a set of Native RPC based APIs. This is the lowest level API'
...
+有了客户端对象之后，我们需要获取 ObjectStore 的引用，获取方法为：
+
+{{< highlight java >}}
+ObjectStore objectStore = ozClient.getObjectStore();
+{{< /highlight >}}
+
+ObjectStore 对象表示了客户端连接的集群。
+
+{{< highlight java >}}
+// 我们创建一个卷来存储数据
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/JavaApi.zh.md]: 我们创建一个卷来存储数据=>让我们创建一个卷来存储数据

File: hadoop-hdds/docs/content/interface/JavaApi.zh.md [hunk 4]
```diff
@@ -21,4 +21,132 @@ summary: Ozone has a set of Native RPC based APIs. This is the lowest level API'
...
+// 使用默认参数来创建 assets 卷
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/JavaApi.zh.md]: assets=》资产

File: hadoop-hdds/docs/content/interface/JavaApi.zh.md [hunk 5]
```diff
@@ -21,4 +21,132 @@ summary: Ozone has a set of Native RPC based APIs. This is the lowest level API'
...
+objectStore.createVolume("assets");
+
+// 验证 assets 卷是否已创建
+OzoneVolume assets = objectStore.getVolume("assets");
+{{< /highlight >}}
+
+
+createVolume 方法也支持接收一个数组来指定卷的创建参数。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/JavaApi.zh.md]: createVolume 方法也支持接收一个数组来指定卷的创建参数=》createVolume 方法也支持传入参数数组来指定卷的创建参数

File: hadoop-hdds/docs/content/interface/JavaApi.zh.md [hunk 6]
```diff
@@ -21,4 +21,132 @@ summary: Ozone has a set of Native RPC based APIs. This is the lowest level API'
...
+
+{{< highlight java >}}
+// 创建客户端对象
+OzoneClient ozClient = OzoneClientFactory.getClient();
+
+// 通过客户端对象获取 ObjectStore 的引用
+ObjectStore objectStore = ozClient.getObjectStore();
+
+// 创建用于存储数据的 assets 卷
+// 此处创建的卷采用默认参数
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/JavaApi.zh.md]: // 用默认参数创建用于存储数据的 assets 卷

Comment (iamabug) [reply hadoop-hdds/docs/content/interface/JavaApi.zh.md]: does we need to translate `assets` here, because it was translated in two occurances above ?

**Comment by xiaoyuyao** [comment]
+1, I will merge it shortly.

**Pull request merged**